tests/utils.py

```diff
@@ -410,3 +419,21 @@ def wrapper(*args: _P.args, **kwargs: _P.kwargs) -> None:
                                     f" args {args} and kwargs {kwargs}")
 
     return wrapper
+
+
+def opcheck(op: Union[torch._ops.OpOverload, torch._ops.OpOverloadPacket,
+                      torch._library.custom_ops.CustomOpDef],
+            args: Tuple[Any, ...],
+            kwargs: Optional[Dict[str, Any]] = None,
+            *,
+            test_utils: Union[str, Sequence[str]] = OPCHECK_TEST_UTILS,
+            raise_exception: bool = True,
+            cond: bool = True) -> Dict[str, str]:
+    if not cond:
+        return {}
+    else:
+        return torch.library.opcheck(op,
+                                     args,
+                                     kwargs,
+                                     test_utils=test_utils,
+                                     raise_exception=raise_exception)
```

Comment:
why do we need this?

Reply:
I used a similar opcheck helper in my other PR.  It was mostly because of the flaky `test_aot_dispatch_dynamic` test.  I didn't want to have to manually specify this every place I used it.

The filter is for making it easier to include opcheck tests wherever the main op is being tested but skip certain combinations so the tests aren't ridiculously slow and/or run out of memory.